doc/man3/X509_verify_cert.pod

```diff
@@ -48,6 +49,11 @@ Both functions return 1 if a complete chain can be built and validated,
 otherwise they return 0, and in exceptional circumstances (such as malloc
 failure and internal errors) they can also return a negative code.
 
+If a complete chain can be built and validated both functions return 1.
+If the certificate must be rejected on the basis of the data available
+or required certificate status data in not available they return 0.
```

Comment:
_or **the** required certificate_

Reply:
I've changed this to: 

or **any** required certificate status data **is** not available

(fixing also a typo) to reflect also the typical case that cert status data is not required (because no CRL checks or the like are requested).